### PR TITLE
[DEV-5471] Unbreak tests

### PR DESCRIPTION
**Pull request by HaraldNordgren**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,10 +6,10 @@ notifications:
...
 
 go:
-    - 1.7
-    - 1.8
-    - 1.9
-    - 1.10
+    - 1.7.x
+    - 1.8.x
+    - 1.9.x
+    - 1.10.x
```
Comment (HaraldNordgren) [inline .travis.yml]: This is necessary becuase otherwise Travis was running Go 1.1

File: Makefile [hunk 2]
```diff
@@ -16,7 +16,7 @@ copyright:
 
 .PHONY: deps
 deps:
-	go get -u github.com/golang/dep
+	curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```
Comment (HaraldNordgren) [inline Makefile]: Dep won't install on versions below 1.8, so make sure to follow the instructions on their readme and install it like this instead.

File: Makefile [hunk 3]
```diff
@@ -37,7 +37,7 @@ lint:
 	             --disable=maligned \
 	             --disable=interfacer \
 	             --disable=varcheck \
-	             --disable=gas \
+	             --disable=gosec \
```
Comment (HaraldNordgren) [inline Makefile]: This is the real change in this ticket :)

File: Makefile (hunk 2 above)
Comment (djui) [reply Makefile]: Could we just drop support for 1.8? (Release: Feb 2017)

Comment (HaraldNordgren) [reply Makefile]: Nooo. Let's keep as many versions as we can unless it becomes a major pain point.

Good open-source projects maintain backward compatibility!

**Pull request merged**